wfe/web-front-end.go

```diff
@@ -624,13 +625,32 @@ func (wfe *WebFrontEndImpl) RevokeCertificate(response http.ResponseWriter, requ
 	}
 }
 
+func (wfe *WebFrontEndImpl) logCsr(remoteAddr string, cr core.CertificateRequest, registration core.Registration) {
+	var csrLog = struct {
+		RemoteAddr   string
+		CsrBase64    []byte
+		Registration core.Registration
+	}{
+		RemoteAddr:   remoteAddr,
+		CsrBase64:    cr.Bytes,
+		Registration: registration,
+	}
+	_, err := json.Marshal(csrLog)
```

Comment:
Actually, I realized that AuditObject already does this for us. If JSON serialization fails, it logs an audit message at the error level with the fmt.Sprintf("%v") of the object. So I removed this marshaling code and now just rely on AuditObject.